prov/psm2/src/makefile

```diff
@@ -0,0 +1,7 @@
+setup:
+	cp ../../psm/src/psmx*.[ch] .
```

Comment:
@jsquyres The next line makes symbol renaming on the files. Symbolic links wouldn't work.